frontend/tsconfig.node.json

```diff
@@ -1,15 +1,16 @@
 {
   "compilerOptions": {
+    "target": "es2017",
     "allowSyntheticDefaultImports": true,
     "composite": true,
     "module": "ESNext",
     "moduleResolution": "bundler",
     "resolveJsonModule": true,
     "skipLibCheck": true,
-    "types": ["axe-playwright"],
+    "types": ["node", "@axe-core/playwright"],
```

Comment:
The "node" types I believe are included by default... Or at least, my tooling is able to recognize built-in node types without this specified here.